simphony/cuds/mesh.py

```diff
@@ -676,6 +684,33 @@ def has_cells(self):
         """
         return len(self._cells) > 0
 
+    def count_of(self, item_type):
+        """ Return the count of item_type in the container.
+
+        Parameter
+        ---------
+        item_type : CUBA
+            The CUBA enum of the type of the items to return the count of.
+
+        Returns
+        -------
+        count : int
+            The number of items of item_type in the container.
+
+        Raises
+        ------
+        ValueError :
+            If the type of the item is not supported in the current
+            container.
+
+        """
+
+        try:
+            return len(self._items_count[item_type]())
+        except:
```

Comment:
>  if I explicitly capture the KeyError exceptions with the actual code. I would have t re-raise them with a different message. We are not doing this anywhere else through the code.

I am not sure why this is a problem. based on the currentl implementation the KeyError is only going to be raised if the item_type is not in self._items_count.